src/drivers/mysql/drv_mysql.c

```diff
@@ -323,10 +323,10 @@ static int mysql_drv_real_connect(db_mysql_conn_t *db_mysql_con)
 
     mysql_ssl_set(con, ssl_key, ssl_cert, ssl_ca, NULL, args.ssl_cipher);
 
-#ifdef MYSQL_OPT_SSL_MODE
```

Comment:
There are many other `#ifdef MYSQL_OPT_SSL_MODE` in this file.

Reply:
OK, this is branch 1.0.

Reply:
Yes, this is a 1.0 PR.